plugin/core/views.py

```diff
@@ -151,12 +151,16 @@ def text_document_identifier(view: sublime.View) -> Dict[str, Any]:
     return {"uri": uri_from_view(view)}
 
 
+def entire_region(view: sublime.View) -> sublime.Region:
```

Comment:
Kinda goes out-of-line with naming? `entire_content_region` would match equivalent function better?

Reply:
Renamed in 62cf41e3b17a91316772fa416a061965a2bd19a9